### ISSUE TITLE
Read set download link seems broken

### ISSUE THREAD

**Issue by skoren** [rrwick/Long-read-assembler-comparison#4]
Hi,

Thanks for hosting and updating this great evaluation. We're interesting in looking at how Canu 2.1 is doing on this data and seeing if we can improve Canu performance. It seems the read set download link is not working right now (https://bridges.monash.edu/articles/dataset/Read_sets/11369952). Both files report:
```
<Error>
<Code>NoSuchKey</Code>
<BucketName>fpfiles</BucketName>
<RequestId>
tx000000000000000001ad1-005f6e0807-148c5119-au-east
</RequestId>
<HostId>148c5119-au-east-au</HostId>
</Error>
```
Using download all aborts after 80 bytes (I assume because the files aren't found). No issues downloading the reference sets. Could you please check if the read links are correct and let us know how to download the data?

Thanks

**Comment by moold** [comment]
Hi, have you solved this problem yet?  I got the same error.

**Comment by rrwick** [comment]
So sorry about this - it's a problem with Figshare/Bridges. I've reported the problem to them (many times) and they assure me they are on it.

In the meantime, somebody else ran into this problem and I scped the files to them, and they kindly made them available on their Dropbox:
https://www.dropbox.com/sh/cfwabn5cywmuq6e/AAC24OU48F-omvj0aHi5KbeBa?dl=0

So give that a try! And I will continue to nag about the broken links 😄
